cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -877,6 +877,14 @@ private void appendRange(StringBuilder query, Range range) {
     appendScanBound(query, range.getEnd(), range.getEndBound().equals(Range.Bound.INCLUSIVE) ? "<=" : "<");
   }
 
+  private void appendSingletonFields(StringBuilder query, Collection<Field<?>> fields) {
+    List<String> conditions = fields.stream()
+      .map(field -> field.getName() + " = ?")
+      .collect(Collectors.toList());
+    
+    query.append(String.join(" AND ", conditions));
```

Comment:
just use `Collections.joining`, no need to make a list

Reply:
updated